DS and Algorithm/SeleniumTest/SeleniumExample.java

```diff
@@ -0,0 +1,38 @@
+// src/main/java/SeleniumExample.java
+public class SeleniumExample {
+    public static void main(String[] args) {
+        // Initialize WebDriver
+        WebDriver driver = WebDriverSetup.initializeChromeDriver();
+
+        // Navigate to the desired website
+        driver.get("https://www.google.com");
+
+        // Perform a search using WebInteraction class
+        WebInteraction.performSearch(driver, "Selenium WebDriver");
+
+        // Wait for a few seconds to see the search results
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
```

Comment:
**Suggestion**: Handle InterruptedException more gracefully by restoring the interrupted status. This approach respects the thread's interrupted status and ensures proper handling of the interrupt. <br> **Code Suggestion**: 
 ```
 - e.printStackTrace();
 + Thread.currentThread().interrupt();
 ```